superset-frontend/src/components/Chart/Chart.jsx

```diff
@@ -232,16 +246,51 @@ class Chart extends React.PureComponent {
     );
   }
 
+  renderSpinner(chartStatus, databaseName) {
+    const messages = {
+      loading: t('Fetching data from %s', databaseName),
```

Comment:
Not sure if we got design input here, but maybe we want to make it more clear that we're waiting for the database to do its job. There's always been this perception of "superset is slow" when in fact it's the underlying datastore that we're waiting on. Maybe a more passive message like "Querying Presto".

Also curious if this is still an overlay (I think previously was transparent?) or just a opaque thing? Maybe a nice effect/animation would be slick here (?) I'm generally not too big on animation, but since this is so common and visible, it may make sense to have something subtle and sharp-looking here.

Reply:
Agree with @mistercrunch here. A passive message will help improve Superset's performance perception.